arch/arm/Kconfig

```diff
@@ -185,6 +185,12 @@ config CPU_HAS_NRF_IDAU
 	  (IDAU: "Implementation-Defined Attribution Unit", in accordance with
 	  ARM terminology).
 
+config CPU_HAS_RENESAS_RA_IDAU
+	bool
+	select CPU_HAS_TEE
+	help
+	  MCU implements the ARM Implementation-Defined Attribution Unit (IDAU).
```

Comment:
no place for this in arch, put in `soc/renesas` folder

Reply:
Indeed have just seen that, @frkv not sure if right person but the nordic parts need to move, can you fix/ping right person to fix?